virtool/labels/api.py

```diff
@@ -21,12 +22,11 @@ async def find(req):
     Get a list of all label documents in the database.
 
     """
-    documents = list()
-    async with AsyncSession(req.app["pg"]) as session:
-        result = await session.execute(select(Label))
-        labels = result.scalars().all()
-        for label in labels:
-            documents.append(label.to_dict())
+    find = req.query.get("find")
```

Comment:
Clashes with the function name. Change to `term` to match other similar handlers (eg. `samples.api.find`)

```suggestion
    term = req.query.get("find")
```